### DISCUSSION TITLE
Any way to run shell scripts from vimium C?

### DISCUSSION THREAD

**Discussion by aflyingpumpkin** [gdh1995/vimium-c#628] [Q&A] [answered]
I would like to quickly be able to open a video in `mpv` or download it with `yt-dlp` both while on the page of the video of by selecting it with linkhints in VimiumC. Is there any way to accomplish this?

**Comment by gdh1995** [comment, marked as answer]
Please see https://github.com/gdh1995/vimium-c/issues/596#issuecomment-1084389663 .

**Reply by aflyingpumpkin** [reply]
Thanks!
